make/Main.gmk

```diff
@@ -108,7 +108,6 @@ ALL_TARGETS += generate-exported-symbols
 #
 # When creating a BUILDJDK, the java targets have already been built and copied
 # into the buildjdk so no need to generate sources.
-ifneq ($(CREATING_BUILDJDK), true)
```

Comment:
Will this regress all other builds?

Reply:
> Will this regress all other builds?

No, there is an internal pipeline build where all platforms built successfully.

Reply:
My concern here is based on the comment just above the removed "ifneq". Will this cause redundant work to be performed and slowdown our builds in a noticeable way? If not, then I'm good with the change.

Reply:
We should probably remove the comment if we're removing the ifneq statement. Can we only remove the ifneq statement on Win/ARM64? Then we don't risk regressing other builds.

Reply:
> We should probably remove the comment if we're removing the ifneq statement. Can we only remove the ifneq statement on Win/ARM64? Then we don't risk regressing other builds.

ah, looks like I cherry picked the wrong commit. I had removed the comment in https://github.com/microsoft/openjdk-jdk11u/pull/44/files.

Reply:
@karianna @JohnTortugo @brianjstafford I have narrowed the scope of the change to cross compiling on Windows.